core/controlplane/config/templates/cloud-config-controller

```diff
@@ -9,6 +9,20 @@ coreos:
     etcd_keyfile: /etc/kubernetes/ssl/etcd-client-key.pem
 
   units:
+{{- range $thisunit := .Controller.CustomUnits}}
```

Comment:
agree, fixed changed the vars to golang convention in all the places you mentioned.